src/ray/gcs/actor_state_accessor.cc

```diff
@@ -90,23 +90,19 @@ Status ActorStateAccessor::AsyncSubscribe(
     const SubscribeCallback<ActorID, ActorTableData> &subscribe,
     const StatusCallback &done) {
   RAY_CHECK(subscribe != nullptr);
-  auto on_subscribe = [subscribe](RedisGcsClient *client, const ActorID &actor_id,
-                                  const std::vector<ActorTableData> &data) {
-    if (!data.empty()) {
-      // We only need the last entry, because it represents the latest state of
-      // this actor.
-      subscribe(actor_id, data.back());
-    }
-  };
+  return actor_sub_executor_.AsyncSubscribe(ClientID::Nil(), subscribe, done);
+}
 
-  auto on_done = [done](RedisGcsClient *client) {
-    if (done != nullptr) {
-      done(Status::OK());
-    }
-  };
+Status ActorStateAccessor::AsyncSubscribe(
+    const ActorID &actor_id, const SubscribeCallback<ActorID, ActorTableData> &subscribe,
+    const StatusCallback &done) {
+  RAY_CHECK(subscribe != nullptr);
+  return actor_sub_executor_.AsyncSubscribe(ClientID::Nil(), actor_id, subscribe, done);
```

Comment:
should use the actual client ID here instead of `ClientID::Nil()`? The current code would subscribe the whole table when we want to subscribe only one actor, is this intended?

Reply:
Got it.  I will try weather random `ClientID` work for actor's subsription.